sys/src/libz/patch/port-inflate.sed

```diff
@@ -1,2 +1 @@
-# avoid signed left shift
-s/-1L << 16/-1UL << 16/g
+/1L.*<<.*16/s//1UL << 16/
```

Comment:
I'm not sure this is right. Shouldn't this convert the input into something like, '~0UL << 16' ?  E.g.,

s/-1L *<< *16/~0UL << 16/

?